README.md

```diff
@@ -33,4 +33,7 @@ npm install
 npm run lint
 npm run build
 npm run dev
+
+Go to http://localhost:3000 and start giving!
```

Comment:
@yahiamarzouk What does "giving" mean in this context?

Reply:
> @yahiamarzouk What does "giving" mean in this context?

Contributing